.github/workflows/main.yaml

```diff
@@ -291,12 +291,74 @@ jobs:
           path: logs-itest-postgres.zip
           retention-days: 5
 
-    # Notify about the completion of all coverage collecting jobs.
+  ########################
+  # Trigger LiTd integration tests
+  ########################
+  trigger-lit-ci:
+    name: trigger LiT itests
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: git checkout
+        uses: actions/checkout@v3
+
+      - name: Setup go environment
+        uses: ./.github/actions/setup-go
+
+      - name: Get current Tap commit hash and branch name
+        run: |
+          echo "TAP_COMMIT=$(git rev-parse HEAD)" >> $GITHUB_ENV
+          echo "TAP_BRANCH=${GITHUB_REF_NAME}" >> $GITHUB_ENV
+
+      - name: Clone Lit repository
+        run: |
+          git clone https://github.com/lightninglabs/lightning-terminal.git
+          cd lightning-terminal
+          git config user.name "github-actions"
+          git config user.email "github-actions@github.com"
+
+      - name: Update go.mod to use the local Tap repository
+        run: |
+          cd lightning-terminal
+          go mod edit -replace=github.com/lightning-labs/taproot-assets=../
+          go mod tidy
+
+      - name: Push changes to a temporary branch in Lit
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          cd lightning-terminal
+          git checkout -b tap-${{ env.TAP_BRANCH }}
+          git commit -am "Update Tap dependency for branch ${{ env.TAP_BRANCH }}"
+          git remote set-url origin https://github-actions:${GITHUB_TOKEN}@github.com/lightninglabs/lightning-terminal.git
```

Comment:
I don't think we actually need or want any changes in `litd`. We just want to know that any changes in `tapd` break the itests, so just checking them out and running them is enough. No need to create a commit in the repo or triggering the CI there. Just run them here directly after the local replace.

Reply:
I know this contradicts what's being described in the issue. But I think issues should describe desired outcome, not actual solutions. Because what we want is that we know tapd changes don't break channels. We can achieve that without pushing commits to litd.

Reply:
ah - this is just pushing a branch (temporarily) to lit's repo so that we can trigger a CI on lit's side that runs on that branch. The branch is deleted in the end

Reply:
> Just run them here directly after the local replace.

ah ok i can look into if pulling in a remote CI in place is possible 👍

Reply:
ok cool - it looks like we can extract the litd itest & unit tests to make them re-usable workflows & then sort of import them here

Reply:
Why can't we just change into the git cloned directory and run `make` commands?

Reply:
ah! good idea - testing locally now 👍